src/components/Draggable/types/ItemTypes.ts

```diff
@@ -0,0 +1,4 @@
+export enum ItemTypes {
```

Comment:
```suggestion
export enum DraggableItemType {
```
The enum names pattern is singular